packages/viz/src/lib/layer/LayerInteractivity.ts

```diff
@@ -180,13 +181,19 @@ export class LayerInteractivity {
 
     let clickStyleProps = {};
 
-    if (this._clickStyle) {
+    if (this._clickStyle === 'default') {
+      const defaultHighlightStyle = this._getDefaultHighlightStyle();
```

Comment:
🆗 I think we can keep both styles the same now, both in yellow for 'hovered' and 'clicked' features. We might decide to change that in the future... thoughts @borja-munoz ?

Reply:
Agree. Let's keep it like this for now.